src/components/resources-section/resource-sidebar/ResourceSidebar.js

```diff
@@ -0,0 +1,18 @@
+import { Sidebar } from 'primereact/sidebar';
+import LinkButton from '@components/learning-units-section/link-button/LinkButton';
+import { Card } from 'primereact/card';
+import AddEvaluation from '@components/resource-section/add-evaluation/AddEvaluation';
+
+const ResourceSidebar = ({ visible, onHideHandler, activeResource, formOptions }) => {
+  return (
+    <Sidebar visible={visible} position="right" onHide={() => onHideHandler()}>
+      <h1>{activeResource.name}</h1>
+      <Card title="Evaluación promedio">
+        {parseFloat(activeResource.average_evaluation).toFixed(1)} <i class="pi pi-star-fill"></i>
```

Comment:
Consider using "Average" component. It fails showing "NaN" instead of "Sin Evaluación".

Reply:
Buena!